plugins/headlamp-plugin/bin/headlamp-plugin.js

```diff
@@ -230,12 +209,64 @@ function extract(pluginPackagesPath, outputPlugins) {
  * @returns {0} Exit code, where 0 is success.
  */
 function start() {
+  /**
+   * Copies the built plugin to the app config folder ~/.config/Headlamp/plugins/
+   *
+   * Adds a webpack config plugin for copying the folder.
+   */
+  function copyToPluginsFolder(webpackConfig) {
+    const packageJson = JSON.parse(fs.readFileSync('package.json', 'utf8'));
+
+    // @todo: should the whole package name be used here,
+    //    and the load be fixed to use? What about namespace packages?
+    const packageName = packageJson.name.split('/').splice(-1)[0];
+    const paths = envPaths('Headlamp', { suffix: '' });
+    const configDir = fs.existsSync(paths.data) ? paths.data : paths.config;
+
+    webpackConfig.plugins = [
+      new FileManagerPlugin({
+        events: {
+          onEnd: {
+            copy: [
+              {
+                source: './dist/*',
+                destination: path.join(configDir, 'plugins', packageName),
+              },
+            ],
+          },
+        },
+      }),
+    ];
+  }
+
+  /**
+   * Inform if @kinvolk/headlamp-plugin is outdated.
+   */
+  function informIfOutdated() {
+    console.log('Checking if headlamp-plugin is up to date...');
+    const outdated = getNpmOutdated();
+    if ('@kinvolk/headlamp-plugin' in outdated) {
+      // const latest = outdated['@kinvolk/headlamp-plugin'].latest;
+      // const url = `https://github.com/kinvolk/headlamp/releases/tag/headlamp-plugin-${latest}`;
+      // const url = `https://github.com/kinvolk/headlamp/releases`;
+      console.warn(
+        '    @kinvolk/headlamp-plugin is out of date. Run the following command to upgrade \n' +
+          //  `    See release notes here: ${url}` +
```

Comment:
I guess the commented out code should be removed?

Reply:
Do you think the message is ok? Or should I add the release notes in there too?

Reply:
I think that having the URL would be more complete.

Reply:
Thanks. Yeah, agreed. I added the release URL in the message.